.env

```
@@ -1,7 +1,7 @@
 EXPO_PUBLIC_MAPBOX_KEY=pk.eyJ1IjoiaW5vdmFjaWVicmF0aXNsYXZhIiwiYSI6ImNrd25icnBvZTJpZDAyb252N2NlcjZvYTEifQ.RYAkWitxV2HyomaXVe4VnA
 EXPO_PUBLIC_CLIENT_ID=f66c44fa-85cc-417d-9de6-be350847a240
 EXPO_PUBLIC_TENANT_ID=fe69e74e-1e66-4fcb-99c5-58e4a2d2a063
-EXPO_PUBLIC_IMAGE_CDN=https://cdn-api.bratislava.sk/enforcement-dev/
+EXPO_PUBLIC_IMAGE_CDN=https://enforcement-dev.s3.bratislava.sk/
```

Looks like these envs are pointing to staging so I guess we want to point to stage bucket as well
```suggestion
EXPO_PUBLIC_IMAGE_CDN=https://enforcement-stage.s3.bratislava.sk/
```